action-bump-version/action.yaml

```diff
@@ -0,0 +1,60 @@
+name: Bump version
+description: Bump the current version in file VERSION and open a new PR with the changes
+
+inputs:
+  current-version:
+    required: true
+    description: the current version to bump
+  branch:
+    required: true
+    description: branch name for the new PR to open
+  level:
+    description: 'what type of version to bump (major, minor, ...), default=patch'
+    default: patch
+
+runs:
+  using: "composite"
+  steps:
+    - name: Ensure git is installed
+      shell: bash
+      run: |
+        git --version
+    - name: Ensure gh is installed
+      shell: bash
+      run: |
+        gh --version
```

Comment:
```suggestion
```

Reply:
nope, it's always installed in github runners, but running in a docker, can make gh not installed.